package.json

```diff
@@ -43,21 +43,21 @@
     "vscode-languageserver-textdocument": "^1.0.0"
   },
   "devDependencies": {
-    "@types/tape": "^4.0.0",
+    "@types/node": "*",
```

Comment:
Would it make sense to pin the current LTS major version range?
Currently `^20.0.0`

Reply:
For dependencies I think `*` makese sense. This allows the package manager to better deduplicate it.

For dev dependencies the same argument could be used, but it’s less important. Also `*` will show when the types break for some reason due to an updated.

An argument for using `^20.0.0` would be that it’s less likely to break builds. This is pretty much the same argument as for using a lockfile.

Another take could be to use the lowest supported Node.js version (`^14.0.0` in this case) to ensure backwards compatibility. However, that doesn’t support `node:test` yet.

I don’t really think there’s a right or wrong here.

Reply:
I'm okay leaving it.
I still have a mild preference for setting a range, for similar reasons to those you list.
But it's a non blocker.

Reply:
Hmm, I also don’t want to make a point of it here.
It’s a project Remco mostly works on, so feel free to do things the way that works for you!

But I’d personally don’t do it, and I’d probably vote against introducing it everywhere else.

> This allows the package manager to better deduplicate it.

I have my doubts. Other tools are going to specify versions, probably. I don’t think npm is running into problems understand the difference between `^20` and `*`